src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -110,5 +112,22 @@ public Task BytesMetrics()
 
             return Task.CompletedTask;
         }
+
+        [RavenAction("/databases/*/periodic-export-stats", "GET")]
```

Comment:
What do you mean null indication? If there isn't a value here?
Where is this used?